### DISCUSSION TITLE
Custom shortcut to place with negative X:Y values

### DISCUSSION THREAD

**Discussion by jcwils** [rxhanson/RectanglePro-Community#276] [Ideas]
Is it possible to enter negative values for X:Y coordinates in a custom shortcut to move windows?
It's useful to hide the status bars of some windows off the bottom of the screen. I can drag them there manually, but it would be nice to have the process automated. Thanks

**Comment by rxhanson** [comment]
Thanks for the feedback. There is a limitation in macOS that prevents windows from programmatically being set partially off the bottom of the screen (there's a discussion for this in Rectangle's repo as well).
